api/server.spec.js

```diff
@@ -18,14 +19,16 @@ describe('/ [GET]', () => {
         expect(res.body.message).toEqual(
           'The Where-to-Code Server is up!',
         );
+        done()
       }));
 
-  it('catches and handles bad request', () =>
+  it('catches and handles bad request', (done) =>
```

Comment:
For a single argument, you can remove the parenthesis around the argument.

Reply:
Done. Prettier has fixed it on format